cns/restserver/api_test.go

```diff
@@ -121,6 +121,14 @@ var (
 		podNamespace: "testpodnamespace",
 	}
 	ncDualNicParams = []createOrUpdateNetworkContainerParams{nc3, nc4}
+
+	// upper-case NCIDs
+	upperCaseNCID1             = "Swift_89063DBF-AA31-4BFC-9663-3842A361F188"
+	upperCaseNCID2             = "Swift_89063DBF-AA31-4BFC-9663-3842A361F189"
+	upperCaseNCIDs             = []string{upperCaseNCID1, upperCaseNCID2}
+	nmaProgrammedNCVersionStr1 = "1"
+	nmaProgrammedNCVersionStr2 = "2"
+	ncVersionList              = map[string]string{}
```

Comment:
Can you move all of this into the test? Using package variables in tests creates inter-test dependencies that make them difficult to debug when the test execution order changes.